api/test/opentelemetry/trace/tracer_provider_test.rb

```diff
@@ -10,9 +10,10 @@
   let(:tracer_provider) { OpenTelemetry::Trace::TracerProvider.new }
 
   describe '.tracer' do
+    # TODO: This test doesn't really do anything for the API tracer
```

Comment:
That's intentional 😄 - it just tests against the spec.

Reply:
Oh excellent; I'll remove the TODO. :D